user-journey.adoc

```diff
@@ -65,51 +94,55 @@ ExecStart=
 ExecStart=/usr/bin/dockerd -H tcp://0.0.0.0:2376 -H unix:///var/run/docker.sock --label provider=generic --insecure-registry 172.30.0.0/16
 Environment=
 ----
-... Run the following commands
+.. Run the following commands:
 +
 ----
 sudo systemctl daemon-reload
 sudo systemctl restart docker
 ----
-.. Mac OSX
-... Configure a CentOS VM using as Hypervisor virtualbox or xhyve. The system will require at least 6 Gb of RAM and 4 CPUs - Process has been documented here: https://gist.github.com/cmoulliard/039e85e17ee99ad8c9e25b975d3f3a8a
-... Next, the docker daemon and Red Hat certificates should be installed using the same instructions for CentOS/Fedora using `yum` or `dnf`.
-. *Determine the docker daemon `ETHERNET_IP_ADDRESS`*
-+
-The ETHERNET_IP_ADDRESS, as defined within this doc, is the ethernet IP accessible from the host to the VM. This address could be part of a private network created behind the host and the guest = vm. It can also be the address of the docker daemon network adapter if running natively.
-+
-TIP: If, the host address of your MacOS, Window or Linux desktop is “192.168.99.20” and the Centos Linux VM running the docker daemon is “192.168.99.50”, then, in this case, the ETHERNET_IP_ADDRESS is “192.168.99.50”
-+
-Refer to https://dzone.com/articles/discovering-dockers-ip-address[Discovering docker daemon’s IP Address] if you need more guidance in determining the correct IP address in a native installation.
 
-== Use case scenario : `git flow`
-. Install istio `oc` client and adds it to your path.
+. *Mac OSX*
+.. Configure a CentOS VM using a Hypervisor virtualbox or xhyve. The process has been documented here: https://gist.github.com/cmoulliard/039e85e17ee99ad8c9e25b975d3f3a8a
+.. Next, install the Docker daemon and Red Hat certificates using `yum` or `dnf` following the same instructions given for Fedora/CentOS.
+
+=== Determine the docker daemon `ETHERNET_IP_ADDRESS`
+
+The ETHERNET_IP_ADDRESS, as defined within this document, is the ethernet IP address accessible from the host to the VM. This address could be part of a private network created behind the host and the guest = vm. It can also be the address of the docker daemon network adapter if running natively.
+
+TIP: If, the host address of your MacOS, Windows or Linux desktop is “192.168.99.20” and the CentOS Linux VM running the docker daemon is “192.168.99.50”, then, in this case, the ETHERNET_IP_ADDRESS is “192.168.99.50”
+
+If you need more guidance in determining the correct IP address in a native installation, refer to https://dzone.com/articles/discovering-dockers-ip-address[Discovering docker daemon’s IP Address].
+
+
+== Continuous Delivery workflow
+
+. Install the istio `oc` client and add it to your path.
```

Comment:
This (istiooc/oc installation and cluster-up step) could actually be separated into a prerequisite step before either of the flow. I didn't notice this before but it's really necessary for both and there's no real reason (other than the github-username & github-api-token..etc parameters for git flow) to have this specific to the git flow.

Reply:
Since we're following the first seven steps for both flows, I'm not going to break it out at this time.